mplc/multi_partner_learning/fast_mpl.py

```diff
@@ -575,3 +575,95 @@ def fit_epoch(model, train_dataset, partners_grads, smodel_list, global_grad, ag
                 break
 
         self.log_end_training()
+
+
+class FastFedGDO(FastFedAvg):
+    """
+     This method is inspired from Federated gradient, but with modification on the local computation of the gradient.
```

Comment:
I would add that FedGDO stand for Federated averaging with double optimizers.

Reply:
👍